src/configuration/evars.md

```diff
@@ -55,6 +55,63 @@ Selects a target operating system of currently loaded binary. Usually, OS is aut
 
 If defined to "true", disassembler view will have flags column.
 
+### asm.tabs
+
+If your disassembly doesn't fit in your screen or aligns clumsily, `asm.tabs` might be of help. You can use this variable to control the distance between the operands, by setting the number of spaces, while displaying your disassembly.
+
+For example, this is the disassemly with the variable set to 0:
+```
+[0x000040a7]> pd 6
+│           0x000040a7      push rsp
+│           0x000040a8      mov r12d, edi                              ; argc
+│           0x000040ab      push rbp
+│           0x000040ac      mov rbp, rsi                               ; argv
+│           0x000040af      push rbx
+│           0x000040b0      sub rsp, 0x48
+```
+And here's what it would look like after setting it to 6:
+```
+[0x000040a7]> e asm.tabs = 6
+[0x000040a7]> pd 6
+│           0x000040a7      push  rsp
+│           0x000040a8      mov   r12d, edi                            ; argc
+│           0x000040ab      push  rbp
+│           0x000040ac      mov   rbp,  rsi                            ; argv
+│           0x000040af      push  rbx
+│           0x000040b0      sub   rsp,  0x48
+```
+
+### asm.tabs.once
+
+This is a boolean variable that can be set to true if you want to align only the opcodes, excluding the arguments.
```

Comment:
In this example I would also show that you set `asm.tabs` or at least say it in the description. Because it is meaningless without `asm.tabs`.  I would also separate it to two snippets with a connecting sentence in the middle, as you showed in `asm.tabs`. The description helps the reader to understand what they are seeing

Reply:
I agree. I've added a line for that.
Please request changes, if anything needs to be rephrased :)